NorthstarDLL/NorthstarDLL.vcxproj

```diff
@@ -102,7 +102,7 @@
       <OptimizeReferences>true</OptimizeReferences>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <EnableUAC>false</EnableUAC>
-      <AdditionalDependencies>$(ProjectDir)include\MinHook.x64.lib;$(ProjectDir)include\libcurl\lib\libcurl_a.lib;dbghelp.lib;Wldap32.lib;Normaliz.lib;version.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>$(SolutionDir)include\MinHook.x64.lib;$(SolutionDir)include\libcurl\lib\libcurl_a.lib;dbghelp.lib;Wldap32.lib;Msimg32.lib;Normaliz.lib;version.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Why do we need msimg32?

Reply:
Needed for `TransparentBlt` function.
https://learn.microsoft.com/en-us/windows/win32/api/wingdi/nf-wingdi-transparentblt